angular-frontend/src/assets/auth.conf.json

```diff
@@ -0,0 +1,11 @@
+{
+  "authority": "http://localhost:9000",
+  "clientId": "angular-frontend",
+  "postLogoutRedirectUri": "http://localhost:4200/",
```

Comment:
**[question]**: is this property used? It seems to be commented out in authorization service.

Reply:
It's not. I will remove it.